src/test/java/com/cloudbees/jenkins/support/CheckFilterTest.java

```diff
@@ -219,24 +226,25 @@ private static String getFirstEnvVar() {
 
     private static class FileChecker {
         private Set<FileToCheck> fileSet = new HashSet<>();
+        private Set<FileToCheck> unchecked = new HashSet<>();
         private Set<String> words = new HashSet<>();
 
         private FileChecker(Jenkins jenkins) throws UnknownHostException {
             fileSet.add(of("manifest.md", "about", false));
             fileSet.add(of("about.md", "b", false));
-            fileSet.add(of("items.md", "jobs", false));
+            // fileSet.add(of("items.md", "jobs", false));
             // checksum.md5 is not generated in tests
 
             // AboutJenkins -> nodes.md
             // The agent node name should be filtered
             fileSet.add(of("nodes.md", AGENT_NAME, true));
             // AboutUser
-            fileSet.add(of("user.md", "anonymous", true));
```

Comment:
Never was used before. Now using `unchecked` to catch such mistakes in the test.